packages/dds/tree/docs/storage/treeStorage-legacy.md

```diff
@@ -0,0 +1,465 @@
+# Shared Tree Storage (Legacy document)
```

Comment:
Why keep this file? If there's value in it still, then let's capture that in the doc. Otherwise, it's in the history :)

Reply:
Eehhh. It's hard to dig things out of the history sometimes... but okay. I suppose very few people would want to reference this file at all and they can copy it and keep it around locally.